src/App.tsx

```diff
@@ -1,26 +1,95 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+import React, { useEffect, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-import { Loader, TodoFilter, TodoList, TodoModal } from './components';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { TodoModal } from './components/TodoModal';
+import { Loader } from './components/Loader';
+import { useTodos } from './hooks/useTodos';
+import { filteringTodos } from './utils/FIlteringTodos';
```

Comment:
The import path './utils/FIlteringTodos' seems to have a typo. Ensure the file name matches exactly, including case sensitivity, to avoid module not found errors.